packages/side-nav/src/presenters/ExternalLinkIcon.js

```diff
@@ -1,24 +1,14 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import { ExternalLink16, ExternalLink24, sizes as iconSizes } from "@hig/icons";
+import { ExternalLinkUI } from "@hig/icons";
```

Comment:
- The new ExternalLink24/16 are so different with previous ones, I don't think they should be used here. And the new ExternalLinkUI in UI icons are most same with previous ExternalLink24. Thus I use `ExternalLinkUI` here.

- As we only has one size for ExternalLinkUI icon, thus, i removed the sizes prop for ExternalLinkIcon. Is this ok?

Reply:
Hey @garyps I talked to design about this and we should got with ExternalLink 24/16. We are actually going to delete the ExternalLinkUI icon, so we can also keep the sizing logic.

Reply:
Thanks @wmui51 I've changed the external link icon to `ExternalLink24/16`. And also made `Link`, `Submodule`, `Title` to be switching 24/16 icons for high/medium density